monkestation/code/modules/blood_for_the_blood_gods/slasher/slasher_datum.dm

```diff
@@ -83,7 +83,7 @@
 
 	RegisterSignal(current_mob, COMSIG_LIVING_LIFE, PROC_REF(LifeTick))
 	RegisterSignal(current_mob, COMSIG_LIVING_PICKED_UP_ITEM, PROC_REF(item_pickup))
```

Comment:
This signal really should be COMSIG_MOB_EQUIPPED_ITEM - but it can't be currently.

To explain: There exists a quirk with how items are equipped to a slot from the hand (i.e. equipping a belt to the belt slot, by clicking on it). Specifically, COMSIG_MOB_UNEQUIPPED_ITEM isn't signalled before COMSIG_MOB_EQUIPPED_ITEM is, in that situation.

But at the same time, unequipping a item from a slot (i.e. belt to a hand slot) DOES signal COMSIG_MOB_UNEQUIPPED_ITEM then COMSIG_MOB_EQUIPPED_ITEM. So this seems like a bug that needs to be fixed upstream.

So for now, this signal runs when COMSIG_LIVING_PICKED_UP_ITEM is signalled - and does not appear to cause any issues.